app/src/main/res/layout-land/appview_appc_reward.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="5dp"
+    android:background="@color/white"
+    >
+
+  <View
+      android:id="@+id/top_line"
+      android:layout_width="match_parent"
+      android:layout_height="1dp"
+      android:background="#f6f7f9"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      />
+
+  <LinearLayout
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_centerHorizontal="true"
+      >
+    <ImageView
+        android:id="@+id/appcoins_logo"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
+        android:layout_below="@id/top_line"
+        android:layout_marginBottom="15dp"
+        android:layout_marginLeft="22dp"
+        android:layout_marginStart="22dp"
+        android:layout_marginTop="16dp"
+        android:src="@drawable/ic_appcoins"
+        />
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/top_line"
```

Comment:
You are using layout_below in a Linear layout, which is an unsupported attribute for linear layout.